packages/editor/src/plugins/link/Form.js

```diff
@@ -46,13 +48,26 @@ const Form = ({ node, editor }) => {
             />
           </Label>
           <Button>Apply</Button>
+          <Button
+            css={css({
+              bg: 'red',
+              color: 'white',
+              ml: 2
+            })}
```

Comment:
We could probably add a secondary button style to the UI components for stuff like this